repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -69,6 +69,8 @@ class KeystoneRequestHandler(identityServiceUri: String, akkaServiceClient: Akka
       )
     )
 
+    // TODO: add tracing info here and add it to trace guid
```

Comment:
Discussion point:  Currently, there are 3 trace points that have been configured: 

* `PowerFilter`
* `RequestProxyServiceImpl`
* `AkkaServiceClientImpl`

`PowerFilter` is a wrapper around all of Repose and the trace point there (although I'd like the entry point to be sooner since there are a number of places we can fail before we emit a trace).

`RequestProxyServiceImpl` also is a fine trace point because it just starts a new span and injects the child headers.

`AkkaServiceClientImpl` I go back and forth.  On one hand, it's good that it's a single point of entry for all filters that need to talk to 3rd party services (Identity, etc.); however, it's not as simple to wrap it in a span since it's an async request that ends before we ever get a response back.  For that, it's better to do it per filter, and it'd be easier to name each child span (e.g. `keystone-filter-span`); however, then we duplicate code and we'd need to remember to add a span for any new filter (including 3rd party filters that may use Repose's akka stuff).

Help!

Reply:
the right spot is to put it down in the http client. that covers everything the akka client does as well as when we go to the origin service, it might even cover the dd (i'd have to look at the code for that one).

Reply:
addressed.